configuration/exodus-pipeline.yaml

```diff
@@ -438,10 +477,113 @@ Resources:
       TimeoutInMinutes: 10
     DependsOn: CodeBuildRole
 
+  IntegrationTestsCodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Condition: IsDev
+    Properties:
+      Name: !Sub ${project}-integration-tests
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+      Source:
+        Type: CODEPIPELINE
+        BuildSpec: |
+          version: 0.2
+          phases:
+            install:
+              runtime-versions:
+                python: 3.7
+              commands:
+                - pip install --upgrade pip
+                - pip install tox
+            build:
+              commands:
+                - git clone https://github.com/$REPO_OWNER/$REPO_NAME.git -b $REPO_BRANCH
+                - cd ./$REPO_NAME
+                - virtualenv integration_tests
+                - source integration_tests/bin/activate
+                - tox -e py37 ./tests/integration -- --cdn-test-url https://$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[0].Outputs[] | select(.OutputKey=="Distribution").OutputValue')
```

Comment:
Can we break the tox command down a little so it's a little easier to read? Like, maybe a variable to hold the URL?

Reply:
Can we put as much as possible into tox.ini instead?

Remember updating the pipeline is more complicated than just merging a commit into the repo.  If we need to change the test command here, like adding or changing the arguments passed to pytest, it would be better if we can just do that by tweaking tox.ini and not having to redeploy the pipeline.

I think ideally it would just be "tox -e integration-tests", if feasible.